plugin.rb

```diff
@@ -16,6 +14,8 @@
 auth_provider icon: "fab-discourse", authenticator: DiscourseLoginClientAuthenticator.new
 
 after_initialize do
+  require_relative "app/controllers/discourse_login_client/auth_controller"
```

Comment:
I don't know enough about Zeitwerk, but we can't use it to autoload here?

Reply:
Possible, I kept having issues with running the specs until I realized it was because of multisites.... Will try without that line